components/resume/themes/DEAFULT_THEME/sections/EducationSection.tsx

```
@@ -25,30 +26,33 @@ const EducationSection = ({section, previousSectionType}: EducationSectionProps)
 				{education.degree && <span>{education.degree}</span>}
-				{education.degree && education.fieldOfStudy && <span> in </span>}
-				{education.fieldOfStudy && <span>{education.fieldOfStudy}</span>}
+				{education.degree && education.field_of_study && <span> in </span>}
+				{education.field_of_study && <span>{education.field_of_study}</span>}
 			</p>
 
 			{/* DESCRIPTION */}
-			{education.description && <p className="text-sm leading-snug mt-0.5 pl-8">{education.description}</p>}
+			{education.description && <div
+				className="text-sm mt-0.5 pl-2 prose prose-sm leading-snug prose-p:m-0 max-w-none"
+				dangerouslySetInnerHTML={{__html: DOMPurify.sanitize(education.description)}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)